EIPS/eip-3374.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3374
+title: Predictable Proof-of-Work (POW) Sunsetting 
+author: @Query0x
+discussions-to: <tbd>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-13
+---
+
+## Simple Summary
+Provides for a predictable, issuance neutral reduction of block rewards to 1 over a one year time span while reducing the uncertainties and risks associated with sudden changes to mining economics.
+
+## Abstract
+Sets the block reward to 3 ETH and then incrementally decreases it every block for 2,362,000 blocks (approximately 1 year) until it reaches 1 ETH.
+
+## Motivation
+Unnecessarily abrupt changes to the Ethereum ecosystem cause disruption and disharmony resulting in the disenfranchisement of community members while undermining stability and confidence.  While moves from Proof-of-Work to Proof-of-Stake will undoubtedly cause friction between those community members vested in either, all benefit from a measured, predictable transition.
+
+This proposal:
+
+1) Is issuance neutral.  There is no overall increase in mining rewards.
+2) Smooths the immediate impact of [EIP-1559](./eip-1559) on mining economics reducing the likeihood of reactionary responses;
+3) Introduces an ongoing, predictable reduction in future mining rewards down to 1, effectively "sunsetting" POW and codifying the move to POS;
+4) Removes economic incentives for continued development of ASICs;
+5) Allows the impacts of decreasing miner rewards to be measured and monitored rather than relying on conjecture and game theory, so adjustments can be made if necessary.
+6) Moves us one step closer to world peace and a more harmonious Ethereum ecosystem.
```

Comment:
While humorous, this line isn't adding value to the EIP so should be removed.

Reply:
Noted... I'll remove it but, for the record, I'm against not striving for world peace and a harmonious Ethereum ecosystem.